### PR TITLE
Update account-functions.adoc with warning of invoke v0 txs to execute

### PR DESCRIPTION
**Pull request by odednaor**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1298/architecture-and-concepts/accounts/account-functions/#attacks_that_execute_limitations_prevent

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1298)

### CODE REVIEW

File: components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc [hunk 1]
```diff
@@ -104,6 +104,11 @@ _Always required_
 
 Initiates the execution stage in the sequencer. The sequencer calls this function upon receiving an `INVOKE` transaction, after the `+__validate__+` function successfully completes.
 
+[WARNING]
+====
+Deprecated `INVOKE` v0 transactions can call the `+__execute__+` function of all contracts. To mitigate this risk, `+__execute__+` calls should explicitly assert that the transaction version is 1 or above.
+====
+
```
Comment (odednaor) [inline components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc:111]: @stoobie not sure if the warning is in the correct format. Would also be good to link INVOKE v0 to this page: https://docs.starknet.io/architecture-and-concepts/network-architecture/transactions/#v0_deprecated_hash_calculation

File: components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc [hunk 2]
```diff
@@ -37,7 +37,7 @@ After successfully completing validation, the deployment is finalized.
 Separating the validation and execution stages guarantees payment to sequencers for work completed and protects them from Denial of Service (DoS) attacks.
 
 [#attacks_that_validation_limitations_prevent]
-== Potential attacks
+== Potential attacks on `+__validate__+`
```
Comment (stoobie) [inline components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc:40]:
```suggestion
== Potential attacks on validation functions
```

File: components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc [hunk 3]
```diff
@@ -77,6 +77,16 @@ So the fees you need to pay to invalidate transactions in the mempool are direct
 ** `get_block_hash`
 ** `get_sequencer_address` `get_sequencer_address` is only supported for Cairo 0 contracts.
 
+[#attacks_that_execute_limitations_prevent]
+== Potential attacks on `+__execute__+`
```
Comment (stoobie) [inline components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc:81]:
```suggestion
== Potential attacks on the `+__execute__+` function
```

File: components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc [hunk 4]
```diff
@@ -77,6 +77,16 @@ So the fees you need to pay to invalidate transactions in the mempool are direct
...
+* Deprecated `INVOKE` v0 transactions can call the `+__execute__+` function of all contracts without going through `+__validate__+ first. 
+* The `+__execute__+` function can be called by any contract.
+
+[#ways_to_mitigate_execute_attacks]
+== Ways to mitigate attacks on `+__execute__+`
```
Comment (stoobie) [inline components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc:86]:
```suggestion
== Ways to mitigate attacks on the `+__execute__+` function
```

File: components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc [hunk 5]
```diff
@@ -77,6 +77,16 @@ So the fees you need to pay to invalidate transactions in the mempool are direct
...
+* Account contracts should explicitly assert in the `+__execute__+` function that the transaction version is 1 or above.
+* Account contracts should explicitly assert in the `+__execute__+` function that the sender is 0 (i.e., the sequencer).
```
Comment (stoobie) [inline components/Starknet/modules/architecture-and-concepts/pages/accounts/account-functions.adoc:88]:
```suggestion
* Account contracts should explicitly assert in the `+__execute__+` function that the sender's address is 0, that is, the sender is the sequencer.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
@odednaor Please see my suggestions

**Comment by odednaor** [comment]
@ArielElp wanted to rewrite this section.
Waiting for his suggestions.

**Comment by odednaor** [comment]
Fixed as par of #1302.

**Pull request closed without merging**